crates/store/re_types/src/any_values.rs

```diff
@@ -0,0 +1,108 @@
+//! Utilities to log arbitrary data to Rerun.
+
+use nohash_hasher::IntMap;
+
+use crate::{
+    ArchetypeFieldName, ArchetypeName, Component, ComponentDescriptor, SerializedComponentBatch,
+};
+use re_types_core::{try_serialize_field, AsComponents, ComponentName, Loggable};
+
+/// A helper for logging arbitrary data to Rerun.
+#[derive(Default)]
+pub struct AnyValues {
+    archetype_name: Option<ArchetypeName>,
+    batches: IntMap<ArchetypeFieldName, SerializedComponentBatch>,
+}
+
+impl AnyValues {
+    /// Assigns an (archetype) name to this set of any values.
+    #[inline]
+    pub fn new(archetype_name: impl Into<ArchetypeName>) -> Self {
```

Comment:
I think most users will first grab for `AnyValues::new()`, and maybe not even realize there is a `AnyValues::default()`, and so they will wonder what they should pick for archetype name.

So I suggest we do builder pattern all the way instead:

```rs
AnyValues::new().with_field(…)
// vs
AnyValues::new().with_archtype_name(…).with_field(…)
```

or alternatively use a more verbose ctor name:


```rs
AnyValues::new().with_field(…)
// vs
AnyValues::from_archtype_name(…).with_field(…)
```

Reply:
I deliberately made that choice to nudge users to provide an archetype name, which should be good practice going forward as it helps with uniquely identifying the data going forward.